src/dns_service_register.cpp

```diff
@@ -162,4 +162,54 @@ NAN_METHOD(DNSServiceRegister) {
     }
 }
 
+NAN_METHOD(DNSServiceUpdateRecord) {
```

Comment:
Could you move this to its own file for consistency?

(I know it's odd and I wouldn't do it again... but that's how it is...)